packages/main/test/specs/DatePicker.spec.js

```diff
@@ -809,13 +809,18 @@ describe("Date Picker Tests", () => {
 		await btnYear.click();
 		let displayedYear = await datepicker.getDisplayedYear(11);
 		assert.ok(await displayedYear.hasClass("ui5-yp-item--disabled"), "Years out of range are disabled");
+	});
+
+	it("Years are disabled when out of range, part 2", async () => {
```

Comment:
why do we need to split the tests?

Reply:
The test would time out and fail otherwise